wandb/sdk/wandb_config.py

```diff
@@ -218,6 +221,28 @@ def update_locked(self, d, user=None, _allow_val_change=None):
         if self._callback:
             self._callback(data=d)
 
+    def merge_locked(self, d, user=None, _allow_val_change=None):
+        num = self._get_user_id(user)
+        callback_d = {}
+
+        for k, v in d.items():
+            k, v = self._sanitize(k, v, allow_val_change=_allow_val_change)
+            self._locked[k] = num
+
+            if (
+                k in self._items
+                and isinstance(self._items[k], dict)
```

Comment:
```suggestion
                isinstance(self._items.get(k), dict)
```
just stylistic suggestion, totally up to you :)